agoradatatools/etl/utils.py

```diff
@@ -30,3 +30,28 @@ def _get_config(config_path: str = None):
         print("Invalid file.  Please provide a valid YAML file.")
         sys.exit(errno.EBADF)
     return config
+
+
+def _find_config_by_name(config: list, name: str):
+    """
+    Iterates through the list to find a dictionary with a key matching 'name'
+
+    Parameters
+    ----------
```

Comment:
Nit: we use trying to change to using Google style docs to be consistent across the entire package. We will slowly make this change in the coming year.

```
Args:
     ....

Returns:
    ....
```

Reply:
Fixed in latest commit.